test/gtest/ucs/test_async.cc

```diff
@@ -471,7 +471,7 @@ UCS_TEST_P(test_async, max_events, "ASYNC_MAX_EVENTS=4") {
 
 UCS_TEST_P(test_async, many_timers) {
     const int max_iters  = 4010 / ucs::test_time_multiplier();
-    const int max_timers = 250 / ucs::test_time_multiplier();
+    const int max_timers = ucs_max(10, 250 / ucs::test_time_multiplier());
```

Comment:
i'd add ucs_max also in line also line 473
(can squash after fix)

Reply:
done